tutorial/part1/chapter2.md

```diff
@@ -1,78 +1,76 @@
-In this section, we initialise an app by creating a Dash instance and calling it "app".
-This one line of code is pretty much static and fixed for any Dash app you may create. In later chapters, we'll talk about external stylesheets and show you how to to specify more properties to create more complex apps. 
+In this section of the code, we initialise the app by creating a Dash instance, tell it to use Bootstrap for the design of the app, and assign it to a variable named "app".
+This one line of code is pretty much static for any Dash app you create. In later chapters, we'll talk more about external stylesheets and show you how to to specify more properties to create more complex apps. 
 
 ### 2.1.3 App Layout
 ```
 app.layout = dbc.Container([
     dcc.Markdown(children='My First App', style={'textAlign': 'center'})
 ])
 ```
-The app layout represents what will be displayed on the web browser. There are a lot of elements that you can include in the app layout, normally they are encapsulated into a "Container". In this minimal example, one single component was added: the dcc.Markdown. This Dash Core Component gives you access to a markup language that makes it easier to format text for web pages. This component has a few properties, such as `children` and `style`:
-- **Children** : this is a common property shared by many Dash components and it allows the adding of textual content. In this Markdown, the "My First App" will be the content displayed on the web page.
+The app layout represents what will be displayed in the web browser. There are a lot of elements that you can include in the app layout, normally they are encapsulated into a "Container". In this minimal example, one single component was added: `dcc.Markdown`. This Dash Core Component let's you use [Markdown syntax](https://commonmark.org/help/) to format the text that is displayed on the page. This component takes has a few properties, such as `children` and `style`:
+- **Children** : this is a common property shared by many Dash components and it allows the adding of textual content. In this Markdown component, "My First App" will be the content displayed on the web page.
 - **Style** : this is another common property shared by many Dash components and defines the look of the component. It requires a dictionary, where the key represents the styling feature you would like to modify, while the value represents how this feature would be modified. In this app, we want to modify the alignment of the text, by centering it.
 
 ### 2.1.4 Run the App
 ```
 if __name__ == '__main__':
     app.run_server()
 ```
-In order to display the app through the browser, we add these statements to "Launch the server". 
+These lines are for running your app. A Dash server is launched which is required to keep the app up and running. The `if` statement makes sure that the app is not launched if we were to import the app as a module and is a Python convention that you don't have have to understand fully.
```

Comment:
...and it's a Python convention that you don't have have (double have) to understand fully.

Or maybe we say something like:
...and it's a Python convention for which an explanation goes beyond the scope of this curriculum.